src/resmom/stage_func.c

```diff
@@ -170,13 +170,11 @@ is_child_path(char *dir, char *path)
 	/* even if the file path is relative to some directory, */
 	/* it may use /../../ notation and escape that directory, */
 	/* so always perform full check of parent-child directory relation */
+	dir_real = realpath(dir, NULL);
+	fullpath_real = realpath(fullpath, NULL);
```

Comment:
And ensure also dir_real and fullpath_real are not NULL on Windows before executing strncpy() into it next...

Reply:
I changed windows version of `realpath` to allocate memory if second argument is NULL. So `dir_real` and `fullpath_real` should not be null after here.

Reply:
Oh OK I see what you mean. `malloc` in `realpath` could still fail.